internal/plugins/ansible/v1/api.go

```diff
@@ -16,17 +16,17 @@ package ansible
 
 import (
 	"fmt"
+	manifestsv2 "github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2"
+	"sigs.k8s.io/kubebuilder/v3/pkg/model/resource"
```

Comment:
`manifestsv2` should be in the last import group and `resource` in the second one.